### ISSUE TITLE
Newline ignored in JsDoc

### ISSUE THREAD

**Issue by linde12** [marijnh/tern#552]
Hello!

Maybe this is just my opinion but i think it would be nice to preserve the newlines in the JsDoc when hinting it. Eg.
```
/**
     * Returns information about the product
     * @params {Object} view Bla bla bla
     * @return {Object} name, View, initialState, save, rollback
     */
```
hints as:
Returns information about the product @params {Object} view Bla bla bla @return {Object} name, View, initialState, save, rollback

It would be nice to get it line-separated.

I'm using Sublime Text 3 and i tried forking the tern_for_sublime project but it seems like this is happening  in tern, not the plugin for sublime.

**Comment by linde12** [comment]
I've solved the problem in `doc_comment.js` but I have to use <br> because the Sublime plugin uses HTML to display the tooltip hint. Do you know of any plugins that work with newline or do they all use HTML?

Because if some client doesn't use HTML then we'll have to do some additional parsing in tern for Sublime(converting \n to <br> before tooltip)

**Comment by marijnh** [comment]
The comment text is passed around as being plain text, so I think you'd have to leave the newlines in, and then have the sublime plugin replace those with `<br>` before displaying.

**Comment by linde12** [comment]
Alright, that's fine!

One more thing i came to realize while writing it up. Regarding removing all the line breaks - isn't it better to keep them? My JsDoc comments can look something like:
```
/**
* This is the start of a very long description and here
* i decide to add a line-break for readability.
* @params ...
**/
```
Even if some plugins have line-wrapping i think it would be nice for the developer to decide where a line-break is suitable. Or is there something i'm missing? Just want to help! :)

**Comment by marijnh** [comment]
Fixed since fd7a9995ff3
